v20.2/manage-a-backup-schedule.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Manage a Backup Schedule
+summary: Learn about how to create, manage, and restore from a schedule of periodic backups.
+toc: true
+---
+
+<span class="version-tag">New in v20.2:</span> You can create schedules in CockroachDB for periodic backups. Once a [backup schedule is created](#create-a-new-backup-schedule), you can do the following:
+
+- [Set up monitoring for the backup schedule](#set-up-monitoring-for-the-backup-schedule)
+- [View and control the backup schedule](#view-and-control-the-backup-schedule)
+- [View and control a backup initiated by a schedule](#view-and-control-a-backup-initiated-by-a-schedule)
+
+## Create a new backup schedule
+
+To create a new backup schedule, use the [`CREATE SCHEDULE FOR BACKUP`](create-schedule-for-backup.html) statement:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE SCHEDULE schedule_name
+  FOR BACKUP INTO 's3://test/backups/test_schedule_1?AWS_ACCESS_KEY_ID=123&AWS_SECRET_ACCESS_KEY=123'
+    WITH revision_history
+    RECURRING '@daily'
+    WITH SCHEDULE OPTIONS first_run = '2020-09-15 00:00:00.00';
+~~~
+
+In this example, a schedule called `schedule_name` is created to take daily backups with revision history in AWS S3, with the first backup being taken at midnight on September 15, 2020.
```

Comment:
Callout on where we're inconsistent on `label` vs `name`

Also, that it's very TECHNICALLY two schedules -- one for incremental and one for full -- that have the same label so that you can query them both. Unclear how pedantic we want to be on that.